src/components/NavHeader.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom';
+import '../styles/NavHeader.css';
+import React from 'react';
+// import { NavLink } from 'react-router-dom';
+// import planetImg from '../public/planet2.png';
+
+const NavHeader = () => {
+  const urls = [
+    {
+      id: 1,
+      url: '/rocket',
+      text: 'Rocket',
+    },
+    {
+      id: 2,
+      url: '/mission',
+      text: 'Missions',
+    },
+    {
+      id: 3,
+      url: '/Profile',
+      text: 'My Profile',
+    },
+  ];
+
+  return (
+    <nav className="navbar">
+      <img src="3212580.png" className="PlantImg" alt="Planet-img" />
+      <h1 className="nav-heading">Space Travelers Hub</h1>
+
+      <ul className="nav-main">
+        {urls.map((url) => (
+          <li className="nav-items" key={url.id}>
+            <Link to={url.url}>{ url.text }</Link>
+          </li>
```

Comment:
It's a project requirement to use Navlink rather than a plane link from react-router-dom. Please consider replacing the link with a nav link as requested by the previous code reviewer (CR) <NavLink /> needs to be used for navigation links
and Remember the active page should be indicated with an underline.

## A general code for your guidance:-

You can have a look at the following code for understanding the instructions better. Remember its not your code but general code with the use of Navlink.

```
import React from 'react';
import { NavLink } from 'react-router-dom';

const NavLinks = () => (
  <nav className="nav-bar">
    <ul className="nav-var-list">
      <li>
        <NavLink className="link" to="/">Rockets</NavLink>
      </li>
      <li>
        <NavLink className="link" to="/Missions">Missions</NavLink>
      </li>
      <li>
        <NavLink className="link" to="/Dragons">Dragons</NavLink>
      </li>
      <li>
        <NavLink className="link separator" to="/Profile">My Profile</NavLink>
      </li>

    </ul>
  </nav>
);

export default NavLinks;
```

## IMAGE:-

See the currently active page has an underlined nav item. ✅ 


![image](https://user-images.githubusercontent.com/94223367/231861813-900975c6-e373-44e4-9ec3-c89162c03976.png)